client.go

```diff
@@ -50,9 +50,41 @@ func NewClient(api plugin.API) *Client {
 		Plugin:        PluginService{api: api},
 		Post:          PostService{api: api},
 		Session:       SessionService{api: api},
-		Store:         &StoreService{api: api},
-		System:        SystemService{api: api},
-		Team:          TeamService{api: api},
-		User:          UserService{api: api},
+		Store: &StoreService{
+			api:     api,
+			version: 0,
+		},
+		System: SystemService{api: api},
+		Team:   TeamService{api: api},
+		User:   UserService{api: api},
+	}
+}
+
+func NewClient2(api plugin.API, driver plugin.Driver) *Client {
```

Comment:
@lieut-data - Looking for some guidance on this. In an ideal world, I'd like to break API altogether and avoid having two `Client` constructors and versioning in the `StoreService`. For now, I just added a `NewClient2` to start the discussion.

Let me know how to proceed.

Reply:
Actually, feel free to break this API all you want: we're currently <1.0 on purpose, with the assumption that consumers will upgrade intentionally and deal with the fallout then.

Reply:
Wonderful, thanks!